.drone.yml

```diff
@@ -88,6 +88,12 @@ pipeline:
             matrix:
                 BUILD: AppImage
 
+    documentation:
+        image: nextcloudci/documentation:documentation-5
+        commands:
+            - cd doc
+            - make html
```

Comment:
No need to build it in every step. Maybe add a documentation matrix entry and only build it once.

Reply:
Fixed this one as well 😉